src/Aether/Aether.fs

```diff
@@ -110,39 +110,39 @@ module Compose =
 
 (* Lenses
 
-   Various lenses implemented for common types such as tuples,
-   lists and maps, along with an id lens (which is useful for composing
-   a lens which has not specific "lensing" elements but is implicitly a chain
-   of one or more isomorphisms. Having an id lens enables the root composition. *)
+   Various lenses for common types such as tuples,
+   lists, and maps; along with an id_ lens (which is useful for composing
+   lenses of scalars, chain of one or more isomorphisms, and enabling
+   the root composition. *)
 
 /// Identity lens returning the original item regardless of modifiction
 let id_ : Lens<'a,'a> =
     (fun x -> x), (fun x _ -> x) 
 
-/// First item of a tuple giving a total lens
+/// First item of a tuple given a total lens
```

Comment:
I'm not sure about this change from "giving" to "given". The intention is that this function "gives" (returns, effectively) a lens, rather than takes one, which is what I might assume with "given". Perhaps this could be clearer though?